### PR TITLE
Editorial for LOJ 1187 - Lining up Students

### PR DESCRIPTION
**Pull request by Nasif-Imtiaj**

### CODE REVIEW

**Review by rebornplusplus — CHANGES_REQUESTED** [review]
Highlight the source and should be all okay.

**Comment by rebornplusplus** [comment]
Also, I am closing the other PR (the previous one). You don't have to make a PR every time you do some changes. Simply commit to your branch and that should appear in the PR as well.

File: 1187/en.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+print the only remaining student as he will be in front of the list, in this case 3, which is our ans.
+
+### Coding Approach:
+A brute force solution with the above idea using vector/array can result into TLE, as we will always need to erase an element and shift the rest. But we can easily solve this using PBDS(Policy Based Data Structure).
+
+PBDS: https://codeforces.com/blog/entry/11080
+
+### Code
+```
```
[the remark this answers is not in the document]
Comment (Nasif-Imtiaj) [reply 1187/en.md:57]: Thank you, applied the stuffs you mentioned.

**Review by rebornplusplus — APPROVED** [review]

**Comment by rebornplusplus** [comment]
Thank you for your contribution. :chocolate_bar:

**Pull request merged**